rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -569,6 +569,53 @@ void gui_application::InitializeCallbacks()
 		};
 	}
 
+	callbacks.on_emulation_stop_no_response = [this](std::shared_ptr<atomic_t<bool>> closed_successfully)
+	{
+		Emu.CallFromMainThread([this, closed_successfully]
```

Comment:
missing check for closed_successfully ptr

Reply:
Fixed.